db/openshift.deploy.yml

```diff
@@ -121,6 +128,23 @@ objects:
         requests:
           storage: "${DB_PVC_SIZE}"
       storageClassName: netapp-file-standard
+
+  ### for db major upgrade PVC only  ###
+  - kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: ${NAME}-${ZONE}-${BACKUP_COMPONENT}-${DB_UPGRADE_VERSION}
+      labels:
+        app: ${NAME}-${ZONE}
+    spec:
+      accessModes:
+        - ReadWriteMany
+      resources:
+        requests:
+          storage: "${DB_BACKUP_PVC_SIZE}"
+      storageClassName: netapp-file-standard
+
```

Comment:
Added this PVC for major version upgrade for db dump, separated from regular backup PVC. Which is probably better.
Tried but can't modify "regular backup PVC" for major upgrade purpose seems due to it was already created.